seqlog/consumer.py

```diff
@@ -58,10 +58,10 @@ def flush(self):
 
             self._cancel_auto_flush()
 
-            current_batch = self.current_batch[:]
+            current_batch = [*self.current_batch]
```

Comment:
Turns out it's the fastest way to shallow copy short lists.

Reply:
Thanks - learned something new! 🙂